<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryWidget</class>
 <widget class="QWidget" name="InventoryWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="statusLabel"><property name="text"><string>库存状态: 未检查</string></property></widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item><widget class="QPushButton" name="checkButton"><property name="text"><string>检查库存</string></property></widget></item>
     <item><widget class="QPushButton" name="refreshButton"><property name="text"><string>刷新</string></property></widget></item>
     <item><spacer name="horizontalSpacer"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeType"><enum>QSizePolicy::Expanding</enum></property></spacer></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 